<launch>
	<arg name="draw" default="True"/>
	<arg name="threshold" default="0.3"/>
	<arg name="chvalue" default="3"/>
	<arg name="x" default="0.7"/>
	<arg name="path" default="/home/fole/sawyer_ws/src"/>
	<arg name="path_openpose" default="/home/fole/openpose/models"/>
	<arg name="undistort" default="True"/>
	<arg name="debug" default="False"/>

	<node name="simulator_node" pkg="hands_free" type="start_robot.sh" output="screen">
	</node>

	<node pkg="hands_free" type="Main.py" name="hands_free_node" output="screen" launch-prefix="gnome-terminal --command">
		<param name="draw" value="$(arg draw)" type="bool"/>
		<param name="threshold" value="$(arg threshold)" type="double"/> 
		<param name="chvalue" value="$(arg chvalue)" type="int"/> 
		<param name="x" value="$(arg x)" type="double"/> 
		<param name="path" value="$(arg path)" type="str"/>
		<param name="path_openpose" value="$(arg path_openpose)" type="str"/>
		<param name="undistort" value="$(arg undistort)" type="bool"/> 
		<param name="debug" value="$(arg debug)" type="bool"/> 
	</node>

</launch>

